infrastructure/ansible/roles/dataset_loader/defaults/main.yml

```diff
@@ -4042,3 +4050,18 @@ dl_ds_default_ds_template:
     longDesc: A basic HTTP routed Delivery Service with Anonymous IP Block Enabled
     anonymousBlockingEnabled: true 
     uniqueKey: simple-http-anon-block
+
+# Federations
+dl_ds_default_federation_user: "{{ federation_user }}"
```

Comment:
`federation_user` is undefined.  It looks like it got added to the sample, but not the actual defaults.

Reply:
`federation_user` defined in 343bcbf8ea.